server/test/services/withings/index.test.js

```diff
@@ -11,32 +11,40 @@ const WithingsService = proxyquire('../../../services/withings', {
 });
 
 describe('withingsService', () => {
+  let countSetValueCount = 0;
+  let countGetValueCount = 0;
   const withingsService = WithingsService(
     {
       variable: {
-        getValue: fake.returns(
-          '{' +
+        getValue: function returnValue(key, serviceId, userId) {
+          countGetValueCount += 1;
```

Comment:
You can still use fake, and `assert.callCount(variable.getValue, 8)` to count number of calls.

Reply:
I had some assert and expect